Content.Server/_RMC14/Xenonids/Egg/XenonidEggSystem.cs

```diff
@@ -0,0 +1,89 @@
+using Content.Shared._RMC14.Hands;
+using Content.Shared._RMC14.Marines;
+using Content.Shared._RMC14.Xenonids.Construction;
+using Content.Shared._RMC14.Xenonids.Egg;
+using Content.Shared._RMC14.Xenonids.Parasite;
+using Content.Shared._RMC14.Xenonids.Plasma;
+using Content.Shared._RMC14.Xenonids.Weeds;
+using Content.Shared.Actions;
+using Content.Shared.Buckle.Components;
+using Content.Shared.Database;
+using Content.Shared.DoAfter;
+using Content.Shared.FixedPoint;
+using Content.Shared.Ghost;
+using Content.Shared.Hands.Components;
+using Content.Shared._RMC14.Xenonids.Egg;
+using Content.Shared.Interaction;
+using Content.Shared.Item;
+using Content.Shared.Maps;
+using Content.Shared.Mind;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Popups;
+using Content.Shared.StepTrigger.Components;
+using Content.Shared.StepTrigger.Systems;
+using Content.Shared.Stunnable;
+using Content.Shared.Tag;
+using Content.Shared.Verbs;
+using Robust.Shared.Map.Components;
+using Robust.Shared.Network;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Serialization;
+using Robust.Shared.Timing;
+using Content.Server.Ghost.Roles;
+using Content.Shared._RMC14.Admin;
+
+namespace Content.Server._RMC14.Xenonids.Egg;
+
+public sealed class XenonidEggSystem : EntitySystem
```

Comment:
Should probably call it XenoEggRoleSystem if you don't want to inherit the shared one and split it, just so there's not a xeno egg system and a xenonid egg system